src/Core/Entity/UserSettingRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Repository for User Setting
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Entity;
+
+use Ushahidi\Core\Entity\Repository\EntityGet;
+use Ushahidi\Core\Entity\Repository\EntityExists;
+
+interface UserSettingRepository extends
```

Comment:
I haven't done this everywhere yet but this should probably extend Create and Update Repository interfaces I think

Reply:
where are those?

Reply:
`Ushahidi\Core\Usecase` I think

Reply:
I think we can do that in a separate ticket, can you add one for it and assign it to @jrtricafort so he can track it into a later cycle?